translations/es.json

```diff
@@ -0,0 +1,138 @@
+      "delete": "Elimniar menu",
+      "title": "Nombre del menú",
+      "url": "Link de la receta",
+      "notes": "Notas (número de página del recetario, por ejemplo)",
+      "tags": "Agregar o eliminar etiquetas",
+      "tagsHint": "Agregar o eliminar mediante drag & drop",
+      "assignedTags": "Etiquetas asignadas",
+      "allTags": "Todas las etiquetas",
+      "ingredients": "Ingredientes",
+      "changeAmount": "Cambiar la cantidad",
+      "removeIngredient": "Elimniar ingrediente",
+      "changeIngredient": "Cambiar ingrediente",
+      "addIngredient": "Agregar ingrediente",
+      "save": "Guardar menú"
+    }
+  },
+  "ingredients": {
+    "title": "Crear, editar y eliminar ingredientes",
+    "introduction": "Aquí puede añadir nuevos ingredientes, editar los existentes o eliminar los innecesarios. Los ingredientes constan de un nombre y unidad en la que se miden.",
+    "edit": {
+      "new": "Agregar un nuevo ingrediente",
+      "modalTitle": "Editar ingrediente",
+      "delete": "Eliminar ingrediente y sus apariciones en otras recetas.",
+      "name": "Nombre",
+      "unit": "Unidad",
+      "save": "Guardar ingrediente"
+    }
+  },
+  "tags": {
+    "title": "Crear, editar y eliminar etiquetas",
+    "introduction": "Aquí puedes añadir nuevas etiquetas, editar las existentes o eliminar las innecesarias. Una etiqueta consta de un nombre, un color y un Emoji como icono.",
+    "edit": {
+      "new": "Agregar etiqueta",
+      "modalTitle": "Editar etiqueta",
+      "delete": "Eliminar etiqueta y sus apariciones",
+      "name": "Nombre",
+      "color": "Color",
+      "icon": "Emoji",
+      "save": "Guardar etiqueta"
+    }
+  },
+  "about": {
+    "title": "Sobre Yummyplan",
+    "nav": "Sobre Yummyplan / donaciones",
+    "text": "Yummyplan ha sido desarrollado con Nuxt, Tailwind y Fontawesome y está alojado en Github. Está publicado bajo la licencia MIT. Si se encuentran errores o bugs, o si se quiere desarrollar una nueva característica, se puede crear un pull request.",
+    "donate": {
+      "title": "Donaciones",
+      "text": "Esta herramienta es gratuita y de código abierto. La he creado para que la planificación de menús para toda una semana sea más fácil y accesible. La planificación de menús te ayuda a comprar con menos frecuencia y a reducir el desperdicio de alimentos, que es algo que todos deberíamos al menos intentar. Hoy más que nunca. Si esta herramienta te ha ayudado y te ha ahorrado tiempo, invítame un café en buymeacoffee:",
+      "paypaltext": "O donar a través de Paypal"
+    }
+  },
+  "emoji": {
+    "frequentlyused": "Usado con frecuencia",
+    "people": "Gente",
+    "nature": "Naturaleza",
+    "objects": "Objetos",
+    "places": "Sitios",
+    "symbols": "Símbolos",
+    "open": "Seleciona un emoji"
+  },
+  "categories": {
+    "fruits": "Frutas",
+    "vegetables": "Vegetales",
+    "cannedFood": "Comida enlatada",
+    "frozenGoods": "Comida congelada,
```

Comment:
Here's a missing quote mark

Reply:
You are right Jose. 
Since you are Spanish, what do you think about the translation of "oder-Suche" and "und-Suche" to "Búsqueda 'o' \",\", Búsqueda inclusiva \"&\"" ?

Reply:
I think it's ok, the space given to the hint text  is so little to elaborate in Spanish how the operators work that you can't just make it better.